cob_frame_tracker/src/cob_frame_tracker.cpp

```diff
@@ -282,21 +282,40 @@ void CobFrameTracker::publishTwist(ros::Duration period, bool do_publish)
     error_msg.twist.angular.y = error_rot_y;
     error_msg.twist.angular.z = error_rot_z;
 
-    if (movable_trans_)
+    //T2
+    
+    if (movable_trans_)    
     {
-        twist_msg.twist.linear.x = pid_controller_trans_x_.computeCommand(error_trans_x, period);
-        twist_msg.twist.linear.y = pid_controller_trans_y_.computeCommand(error_trans_y, period);
-        twist_msg.twist.linear.z = pid_controller_trans_z_.computeCommand(error_trans_z, period);
+        if ((fabs(error_msg.twist.linear.x) <= 0.005) && (fabs(error_msg.twist.linear.y) <= 0.005) && (fabs(error_msg.twist.linear.z) <= 0.005))
+        {
+            this->publishHoldTwist(period);
+            ROS_INFO_STREAM("PUBLISHING ZERO SPEED...");
```

Comment:
undo these changes....

there is a better way to do this, i.e. use `checkCartDistanceViolation()` with according parameter
Maybe this criteria is not correctly implemented yet as documented in https://github.com/ipa320/cob_control/issues/75. Thus we should investigate and fix it there in the function